### ISSUE TITLE
Synchronize with nearest camera position

### ISSUE THREAD

**Issue by kllgjc** [Florian-Barthel/splatviz#46]
Hey, love this viewer! One thing that I think would definitely help is to add a wdiget that gives a side-by-side view of the _nearest_ camera image as you zoom and rotate around the splat, and the ability to jump the viewer into that exact camera position as well for a direct comparison.

**Comment by Florian-Barthel** [comment]
I agree, this would be awesome. I will keep this in mind when developing new features. But feel free to add this in a PR, if you have the time
